Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL = test
+.PHONY: test validate respec
+
+VALIDATOR ?= https://validator.w3.org/nu/
+VALIDATOR_OPTS ?= -F laxtype=yes -F parser=html5 -F level=error
+
+curl = curl -sSLf
+respec = respec2html --disable-sandbox -ew
```

Comment:
Is `respec2html` a dependency? If so, how does it get installed and can we document it in case someone wants to run it locally?

Reply:
Fixed in https://github.com/w3c/webdriver/pull/1339/commits/44c7faaad841b299283aa5c2ba353e06a7852341.